Makefile

```
@@ -587,67 +595,61 @@ test-coverage-cpp-standalone: ## Run C++ tests with coverage (standalone)
+benchmark: build-development-image ## Run C++ benchmarks
 
-	@ echo "Running C++ benchmark..."
+	@ echo "Running C++ benchmarks..."
 
 	docker run \
 		--rm \
 		--volume="$(CURDIR):/app:delegated" \
 		--volume="/app/build" \
-		--workdir=/app/build \
+		--workdir=/app \
 		$(docker_development_image_repository):$(docker_image_version) \
```

checkmake (0.3.2)

[warning] Target "$(docker_development_image_repository)" defined multiple times (lines 193 and 609).

(uniquetargets)